src/screens/organizations/index.js

```diff
@@ -1,97 +1,72 @@
-      iconKey: "rethink-priorities",
-      links: {
-        visit: "https://rethinkpriorities.org/",
-        donate: "https://rethinkpriorities.org/donate/",
-        volunteer: "",
-      },
-    },
-    {
-      name: "faunalytics",
-      description:
-        "Faunalytics' mission is to empower animal advocates with access to research, analysis, strategies, and messages that maximize their effectiveness to reduce animal suffering.",
-      iconKey: "faunalytics",
-      links: {
-        visit: "https://faunalytics.org/",
-        donate: "https://faunalytics.org/donate/",
-        volunteer: "",
-      },
-    },
-  ];
-
   return (
-    <>
-      <section id="organizations" className="organizations screen-content">
-        <div className="full-screen h-full w-full flex-col">
-          <div className="split-screen h-screen-50vh flex w-[35%] flex-col items-center justify-center py-16 text-center">
-            <h2>organizations working on wild animal suffering</h2>
-          </div>
-          <div className="split-screen h-screen-50vh relative w-full flex-row flex-wrap justify-around">
-            {organizations.map((organization) => {
-              return (
-                <div
-                  className="flex w-1/2 justify-center"
-                  key={organization.name}
-                >
-                  <div className="flex h-full w-1/2 flex-row">
-                    <div className="flex flex-row p-2">
-                      <div
-                        className={`${organization.iconKey} aspect-square h-20`}
-                      ></div>
-                    </div>
-                    <hr className="orange" />
-                    <div className="flex flex-col">
-                      <h4 className="p-2">{organization.name}</h4>
-                      <div className="p-2">{organization.description}</div>
-                      <div className="flex flex-row p-2">
-                        <span className="badge brown m-1 capitalize">
-                          visit site
-                        </span>
-                        <span className="badge sand m-1 capitalize">
-                          donate
-                        </span>
-                        <span className="badge oat m-1 capitalize">
-                          volunteer
-                        </span>
-                      </div>
-                    </div>
-                  </div>
+    <section
+      id="organizations"
+      className="organizations relative min-h-screen py-24"
+    >
+      <div className="relative mx-auto flex w-full max-w-7xl flex-col space-y-16 px-10 text-white md:px-32 lg:px-10">
+        <h2 className="text-center font-brand text-4xl">
+          Organizations working on wild animal suffering
+        </h2>
+
+        <div className="grid gap-y-12 lg:mx-8 lg:grid-cols-2 lg:gap-x-48 lg:gap-y-16 xl:mx-24">
+          {organizations.map((organization, key) => (
+            <div
+              key={key}
```

Comment:
It's a bad practice to use the index as a key. That's what React does by default already, but it's better to have something truly unique that ties to `organization`. So this should be `key={organization.name}` or something similar.

Reply:
Good point! I looked it up and the reasoning behind it being a bad practice makes sense. Will fix.